shared/styles/mixins.scss

```diff
@@ -14,82 +14,65 @@
   @return $number;
 }
 
-// defaults to standard heading size
-@mixin responsive-font($min-size: ($font-size - 2), $size: $font-size, $limit: $page-limit) {
+@mixin responsive-font($min-size: $font-size-min, $size: $font-size, $limit: $page-limit) {
+
+  $unitless-min: strip-unit($min-size);
+  $unitless-size: strip-unit($size);
+
   $baseline: $min-mobile;
-  $font-multiplier: (($size - $min-size) / (strip-unit($limit) - strip-unit($baseline)));
+  $font-multiplier: (($unitless-size - $min-size) / (strip-unit($limit) - strip-unit($baseline)));
```

Comment:
`$min-size` here, and on line 24, could be `$unitless-min`, right?

Reply:
yep! will fix